src/httpx.nim

```diff
@@ -219,9 +270,19 @@ template acceptClient() =
         return
 
     raiseOSError(lastError)
+
   setBlocking(client, false)
-  selector.registerHandle(client, {Event.Read},
-                          initData(Client, ip = address))
+
+  template regHandle() =
+    selector.registerHandle(client, {Event.Read}, initData(Client, ip = address))
+
+  when usePosixVersion:
+    # Only register the handle if the file descriptor count has not been reached
+    if likely(client.int < osMaxFdCount):
+      regHandle()
```

Comment:
Log if the max is reached? Do you think that might spam the log a bit? My test case was overloading the server with 10k connections at once, I imagine it would log a lot for something like that

Reply:
Fine, it could be added later.